components/ide/jetbrains/backend-plugin/src/main/kotlin/io/gitpod/jetbrains/remote/GitpodCLIService.kt

```diff
@@ -46,9 +52,20 @@ class GitpodCLIService : RestService() {
             if (url.isNullOrBlank()) {
                 return "url is missing"
             }
-            return withClient(request, context) { project ->
-                BrowserUtil.browse(url, project)
+            GlobalScope.launch {
```

Comment:
all client based requests should await client, i.e open as well

Reply:
Cross-posting: https://github.com/gitpod-io/gitpod/pull/10129#discussion_r881477632